platform/web/js/libs/library_godot_display.js

```diff
@@ -84,41 +84,41 @@ const GodotDisplayVK = {
 
 			let elem = GodotDisplayVK.textinput;
 			switch (type) {
-			case 0: // KEYBOARD_TYPE_DEFAULT
-				elem.type = 'text';
-				elem.inputmode = '';
-				break;
-			case 1: // KEYBOARD_TYPE_MULTILINE
-				elem = GodotDisplayVK.textarea;
-				break;
-			case 2: // KEYBOARD_TYPE_NUMBER
-				elem.type = 'text';
-				elem.inputmode = 'numeric';
-				break;
-			case 3: // KEYBOARD_TYPE_NUMBER_DECIMAL
-				elem.type = 'text';
-				elem.inputmode = 'decimal';
-				break;
-			case 4: // KEYBOARD_TYPE_PHONE
-				elem.type = 'tel';
-				elem.inputmode = '';
-				break;
-			case 5: // KEYBOARD_TYPE_EMAIL_ADDRESS
-				elem.type = 'email';
-				elem.inputmode = '';
-				break;
-			case 6: // KEYBOARD_TYPE_PASSWORD
-				elem.type = 'password';
-				elem.inputmode = '';
-				break;
-			case 7: // KEYBOARD_TYPE_URL
-				elem.type = 'url';
-				elem.inputmode = '';
-				break;
-			default:
-				elem.type = 'text';
-				elem.inputmode = '';
-				break;
+				case 0: // KEYBOARD_TYPE_DEFAULT
```

Comment:
Seems like your IDE wrongly changed the style in this file.

Reply:
whoops, will fix it in a min